__tests__/controllers/users.controller.spec.ts

```diff
@@ -4,7 +4,9 @@ import { factory, useSeeding } from 'typeorm-seeding';
 import app from '@app';
 import connection from '@database/connection';
 import { User } from '@entities/user.entity';
-import { createJWT } from '@services/jwt.service';
+import 'reflect-metadata';
```

Comment:
is this line necessary? since it's already imported in the `app`

Reply:
Nope, you're right!